src/Symfony/Component/Notifier/Bridge/LineBot/LineBotOptions.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LineBot;
+
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+use Symfony\Component\Notifier\Notification\Notification;
+
+/**
+ * @see https://developers.line.biz/en/reference/messaging-api/#send-push-message-request-body
+ *
+ * @author Yi-Jyun Pan <me@pan93.com>
+ */
+final class LineBotOptions implements MessageOptionsInterface
+{
+    /**
+     * @param array<string, mixed>[] $messages
+     * @param string[]|null          $customAggregationUnits
+     */
+    public function __construct(
+        private ?string $to = null,
+        private array $messages = [],
+        private ?bool $notificationDisabled = null,
+        private ?array $customAggregationUnits = null,
+    ) {
+    }
+
+    /**
+     * @return array{
+     *     to: string|null,
+     *     messages: array<string, mixed>[],
+     *     notificationDisabled: bool|null,
+     *     customAggregationUnits: string[]|null,
+     * }
+     */
+    public function toArray(): array
+    {
+        return [
+            'to' => $this->to,
+            'messages' => $this->messages,
+            'notificationDisabled' => $this->notificationDisabled,
+            'customAggregationUnits' => $this->customAggregationUnits,
+        ];
+    }
+
+    public function getRecipientId(): ?string
+    {
+        return $this->to;
+    }
+
+    /**
+     * ID of the target recipient.
+     *
+     * @return $this
+     */
+    public function to(string $userId): static
+    {
+        $this->to = $userId;
+
+        return $this;
+    }
+
+    /**
+     * Messages to send. Max: 5.
+     *
+     * @see https://developers.line.biz/en/reference/messaging-api/#message-objects
+     *
+     * @param array<string, mixed> $message the message object to add to
+     *
+     * @return $this
+     */
+    public function addMessage(array $message): static
+    {
+        if ($this->getMessagesCount() >= 5) {
```

Comment:
This condition should probably be removed as we try to avoid replicating upstream constraints as they might change. I suppose that when one uses more than 5 messages, the API returns a proper error message already.